nsv13/code/__HELPERS/names.dm

```diff
@@ -38,3 +38,66 @@ var/list/ship_name_suffix = list("slayer","blazer","bringer","slapper","twirler"
 	var/name = ""
 	name = pick(ship_name_prefix) + pick(ship_name_suffix)
 	return capitalize(name)
+
+/var/list/fighter_noun = list(
+	"Paw","Hand","Warmth","Eye","Wings","Palms","Pillar",
+	"Dawn","Fist","Chicken","Hawk","Sparrow","Thunder","Watcher",
+	"Guardian","Stalker","Heron","Swan","Swallow","The Unladen Swallow",
+	"Wren","Corvid","Falcon","Eagles","Owl","Kestrel","Harrier","Vultures",
+	"Osprey","Parrot","Kingfisher","Toucan","Trogon","Inflation","Union",
+	"Cayenne","Mortgage","Hen","Light","Puma","Aglet","Seller","Steam",
+	"Lighting","Wriggler","Carp","Ginseng","Hyena","Tenement","Jet","Cheese",
+	"Asterisk")
```

Comment:
This is not how you are supposed to do global variables, very bad. 🗞️
you'd use the GLOBAL_LIST_INIT define

Reply:
Done

Reply:
> This is not how you are supposed to do global variables, very bad. 🗞️ you'd use the GLOBAL_LIST_INIT define

Yes instead we use var/static/list inside the proc that generates the names :)